interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,174 @@
+# Integrating with AzureML
+
+## Running a Pipeline in AzureML
```

Comment:
Suggest you list the files needed for this - I also have a comment later which motivates the use of AML pipelines. So we want to lead in with "what are AML pipelines", "why we're using them?", "what's needed - list files and describe them"

Reply:
Updated README.md with a list of files needed in this commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/5fedf1138bf1121526974806af256ac71ca3bfa2 and also added content on AML background and how we are using it in this commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/5aacf791335c4982b6c35f374a7ebaa0748be563